jobs/jobs.yml

```diff
@@ -72,19 +72,27 @@
             python-version: 'Python3.12'
         - '3.11':
             python-version: 'Python3.11'
+        - '3.10':
+            python-version: 'Python3.11'
```

Comment:
Copy-paste error, this mixes 3.10 and 3.11.

Reply:
It should be fixed already, both in code and on jenkins.plone.org itself

Reply:
Thanks. :-)